zanata-war/src/main/java/org/zanata/security/UserRedirectBean.java

```diff
@@ -166,9 +167,13 @@ public boolean isRedirectToRegister() {
         return isRedirectTo(REGISTER_URL);
     }
 
+    public boolean isRedirectLoginPage() {
```

Comment:
`isRedirectToLoginPage` to keep it consistent with the other similar method names.